src/utils/jsonstore.ts

```diff
@@ -2,9 +2,16 @@ class JsonStore {
   set(key: string, value: unknown) {
     localStorage.setItem(key, JSON.stringify(value));
   }
-  get<T>(key: string, defaultValue?: T): T {
+  get<T>(key: string, defaultValue?: T): any {
     if (this.has(key)) {
-      return JSON.parse(localStorage.getItem(key)!);
+      try {
+        return JSON.parse(localStorage.getItem(key)!);
+      } catch (exception) {
+        console.warn(exception);
+        const value = localStorage.getItem(key);
+        this.set(key, value);
+        return value;
+      }
```

Comment:
Didn't add this initially when changing the jsonstore from js to ts in [this commit](https://github.com/searchspring/asanaban/commit/30c27f3776b1ff123c23b88e13ab364ab485d74c#), but Rachel's asanaban wasn't working without it.